ci-operator/config/openshift/assisted-installer/OWNERS

```diff
@@ -0,0 +1,69 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```

Comment:
We don't need all these OWNERS. Just people who handle the CI.

Reply:
I'm not sure what to do. It's auto-generated and specifically says do not edit. Is there another way to handle this?

Reply:
You are right. It seems that's brought from the repository itself. It wasn't like this last time I created a config file. :) 
I guess it's fine.